### ISSUE TITLE
Auto notification if postmark service has issues?

### ISSUE THREAD

**Issue by elbsurfer** [wildbit/pm-slack-app#8]
Hello,
am I right that this Bot does not post into one of my channels in case the postmark service has issues but I can only ask pro-actively how the status is? A notification functionality in case something is wrong on postmark side would be much better, would it? Any plans to integrate this?

**Comment by rianvdm** [comment]
That's correct. Unfortunately that's a limitation of the status page at the moment, we don't have a webhook that posts updates. This is something we want to add in the future for sure.

**Comment by elbsurfer** [comment]
Thanks, that would be very helpful. @rianvdm

**Comment by elbsurfer** [comment]
Thanks @derekrushforth
